### PR TITLE
Better Transitions for Event Cards' buttons

### PR DESCRIPTION
**Pull request by mbad0la**
pacerresolve1

### CODE REVIEW

**Comment by handasolo** [comment]
Thanks for the quick PR @mbad0la. The transitions do seem smoother now. But, there is a reason to keep the location on the cards without hover. It helps to deliver essential information to the user without them having to hover. I know it overflows right now, but that's because of  #54 which #75 will fix. I would ask you to undo hiding the location from the cards. In case, you have  some other reason to do so, please let us know. Thanks.

**Comment by manrajgrover** [comment]
Thanks for the PR @mbad0la. Adding to @Slayslot's comment, please remove commented CSS properties too.

**Comment by mbad0la** [comment]
@Slayslot, i did that to overcome erratic changes in the location's span position. I would suggest changing the location's div inside the info-box to be the third child rather than the fourth. Should i try that? @ManrajGrover , will do.

**Comment by handasolo** [comment]
@mbad0la What erratic changes in the position are your talking about? please elaborate. I see it at the same position.

**Comment by mbad0la** [comment]
@Slayslot , due the dynamic nature of the height of figcaption, whenever you hover out of the card, the location div jumps up to maintain it's style attributes. Anyway, i'm trying to keep locations visible as well as keep the transitions working.

**Comment by handasolo** [comment]
Maybe you can make that jump a delayed transition. Might look good, I'm not so sure.

**Comment by mbad0la** [comment]
I might have got a middle ground by changing the card template a bit. Will send a screenshot here before commiting to my fork.

**Comment by handasolo** [comment]
@mbad0la Great! would love to see.

**Comment by mbad0la** [comment]
@Slayslot 
pacerresolve1
Is this as you expect it to be?

**Comment by handasolo** [comment]
@mbad0la Yup, looks good to me.

**Comment by mbad0la** [comment]
@Slayslot  @ManrajGrover , please review the PR.

File: src/client/css/style.css [hunk 1]
```diff
@@ -567,7 +553,7 @@ section#events {
 .cardfigure span {
   font-weight: bold;
   position: relative;
-  bottom: 4.0em;
+  /*bottom: 4.0em;*/
```
Comment (manrajgrover) [inline src/client/css/style.css]: Nit. Please remove this commented line too.

File: index.html [hunk 2]
```diff
@@ -46,10 +46,11 @@ <h4 class="pull-right"></h4>
                   </div>
                 </div>
               </div>
-              <div class="row">
-                <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
-                  <span></span>
-                </div>
+
```
Comment (handasolo) [inline index.html]: Unnecessary space.

File: src/client/css/style.css [hunk 3]
```diff
@@ -32,11 +32,8 @@ a {
 
 #outer-wrapper {
   position: relative;
-
```
Comment (handasolo) [inline src/client/css/style.css:35]: Put this space back in. This is the work of CssComb. Makes CSS more readable.

File: src/client/css/style.css [hunk 4]
```diff
@@ -32,11 +32,8 @@ a {
...
   min-width: 320px;
   min-height: 100%;
-  /*margin-bottom: 200px;*/
-
```
Comment (handasolo) [inline src/client/css/style.css:39]: Same here.

**Comment by handasolo** [comment]
Looks good to me. Github has this new feature to squash commits from PRs as an admin, I'd love to try that :D. If somehow it doesn't work, you'll need to squash your commits as well.

**Comment by mbad0la** [comment]
@Slayslot , have a look at the changes. I guess it's done now.

**Comment by CuriousLearner** [comment]
LGTM

@Slayslot have a look.

File: src/client/css/style.css [hunk 5]
```diff
@@ -32,11 +32,10 @@ a {
 
 #outer-wrapper {
   position: relative;
-
+
```
Comment (handasolo) [inline src/client/css/style.css]: Trailing space.

**Comment by mbad0la** [comment]
@Slayslot , you told me to put the space back in.

**Comment by handasolo** [comment]
@mbad0la Yes, but not a trailing space.

**Comment by mbad0la** [comment]
@Slayslot , what about now?

**Comment by handasolo** [comment]
Looks good to me now. Thanks for your patience @mbad0la. I'll let you know if you need to squash your commits.

**Comment by mbad0la** [comment]
Sure

**Comment by CuriousLearner** [comment]
Thanks @mbad0la for the PR.

Looks good, merging in :+1:

**Pull request merged**